hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AUtils.java

```diff
@@ -676,7 +712,7 @@ public static AWSCredentialProviderList buildAWSProviderList(
    * @return the instantiated class
    * @throws IOException on any instantiation failure.
    */
-  private static AWSCredentialsProvider createAWSCredentialProvider(
+  private static AWSCredentialsProvider createAWSV1CredentialProvider(
```

Comment:
this is something which could be moved into the .auth module and while built with the v1 sdk, should be something we don't need on the CP except when there's what is (probably) a v1 credential provider listed